gradle/releases.gradle.kts

```diff
@@ -0,0 +1,255 @@
+// Common regex patterns
+private val APP_VERSION_REGEX = """<string name="app_version">([^(]+) \((\d+)\)</string>""".toRegex()
+private val VERSION_HEADER_REGEX = """## Version (.*) \(([a-f0-9]+)\)""".toRegex()
+private val COMMIT_HASH_REGEX = """([a-f0-9]+) -.*""".toRegex()
+private val MERGE_PR_REGEX = """([a-f0-9]+) - Merge pull request #(\d+) from.*""".toRegex()
+
+// Helper function to run git commands
+private fun runGitCommand(vararg args: String): String {
+    val process = ProcessBuilder("git", *args)
+        .directory(projectDir)
+        .redirectOutput(ProcessBuilder.Redirect.PIPE)
+        .start()
+
+    val output = process.inputStream.bufferedReader().readText().trim()
+    process.waitFor()
+    return output
+}
+
+// Function to normalize content by ensuring there's only one empty line between sections
+private fun normalizeContent(content: String): String {
+    return content.replace(Regex("\n{3,}"), "\n\n")
+}
+
+tasks.register("prepareRelease") {
+    group = "releases"
+    description = "Bumps versions and prepares release notes"
+
+    dependsOn("updateVersion", "updateReleaseNotes")
+
+    doLast {
+        // Export library definitions for licenses
+        project.exec {
```

Comment:
This nested Gradle invocation isn't pretty, and I welcome suggestions for improvements, but this is the best I could do for now and it does work fine for the simple thing we're doing here.

Reply:
Curious to hear if there's anything can do on this task to make it easier to integrate for such purposes?

Reply:
Now that you ask, I actually figured out what would be the nicer way for this. Can we configure the exportPath in the shared module's Gradle script somehow? I see file name but not path in the [extension](https://github.com/mikepenz/AboutLibraries/blob/develop/plugin-build/plugin/src/main/kotlin/com/mikepenz/aboutlibraries/plugin/AboutLibrariesExtension.kt).

If we could do that, then this could be a simple task dependency.